### PR TITLE
Add graphviz-based module-module compile dependency graphs

### PR DESCRIPTION
**Pull request by andr3ej**
* `./gradlew graphs` will create the reports assuming graphviz `dot` is installed
* On Linux, it will then launch eog to view the graphs

@Clintonio is this similar to what you've done?

### CODE REVIEW

File: constants.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-gradlePluginsVersion=0.13.3
+gradlePluginsVersion=0.13.3+andr3ej-compile-graphs
```
Comment (Clintonio) [inline constants.properties]: Is this necessary?

Comment (andr3ej) [inline constants.properties]: I think an earlier version of this branch did actually change plugins, so I'll remove this. (It's non-numeric while in branch so a git merge from master doesn't clobber it.)

Comment (andr3ej) [inline constants.properties]: Removed

**Comment by andr3ej** [comment]
Moved to tools.

**Pull request merged**